common/containers/Tabs/Contracts/components/TxCompare.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Wei } from 'libs/units';
+import translate from 'translations';
+import Code from 'components/ui/Code';
+export interface Props {
+  nonce: string;
+  gasPrice: Wei;
+  gasLimit: string;
+  to: string;
+  value: string;
+  data: string;
+  chainId: number;
+  signedTx: string;
+}
+
+export const TxCompare = (props: Props) => {
+  if (!props.signedTx) {
+    return null;
+  }
+  const { signedTx, ...rawTx } = props;
+  const Left = () => (
+    <div className="form-group">
+      <h4>{translate('SEND_raw')}</h4>
+      <Code>
+        {JSON.stringify(
+          { ...rawTx, gasPrice: rawTx.gasPrice.toString(16) },
+          null,
+          2
+        )}
+      </Code>
+    </div>
+  );
+  const Right = () => (
+    <div className="form-group">
+      <h4> {translate('SEND_signed')} </h4>
+      <Code>{signedTx}</Code>
+    </div>
+  );
+  return (
+    <section className="row">
```

Comment:
section doesn't need `className="row"`, removing this fixes mis-aligned margins on the raw and signed transaction when deploying a contract.